packages/klevu-ui/src/components/klevu-quicksearch/klevu-quicksearch.css

```diff
@@ -88,3 +89,13 @@ klevu-moi {
     float: right;
   }
 }
+
+.noResultsMessage {
+  text-align: center;
+  border-bottom: 1px solid var(--klevu-color-border, #d6d6d6);
```

Comment:
There is no such variable as --klevu-color-border. I noticed that there are references to this variable in the codebase. It's legacy. We should use --klevu-color-neutral-3 or --klevu-color-neutral-4 in this case.

Reply:
I created separate task to remove --klevu-color-border from the codebase.